clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -12990,6 +12990,13 @@ def err_acc_duplicate_clause_disallowed
     : Error<"OpenACC '%1' clause cannot appear more than once on a '%0' "
             "directive">;
 def note_acc_previous_clause_here : Note<"previous clause is here">;
+// TODO(OpenACC): Combine these with the one above, and decide between a
```

Comment:
I'll fix this as NFC when this gets merged, but the size of this review would explode if I were to do it now.